builds/misc/addons-release.yaml

```diff
@@ -54,12 +60,23 @@ jobs:
     dependsOn:
       - linux_API_proxy_module
     steps:
+    # Both docker logins needed for if we need to test this job. In this case images should go to edgebuilds.
+    - task: Docker@2
+      displayName: Docker login edgebuilds
+      inputs:
+        command: login
+        containerRegistry: iotedge-edgebuilds-acr
+    - task: Docker@2
+      displayName: Docker login edgerelease
+      inputs:
+        command: login
+        containerRegistry: iotedge-release-acr,
     - bash: |
           if [ -z '$(version)' ]; then
             echo '##vso[task.setvariable variable=buildVersion]$(Build.BuildNumber)'
           else
             echo '##vso[task.setvariable variable=buildVersion]$(version)'
           fi
       displayName: 'Set build version'
-    - script: scripts/linux/buildManifest.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(buildVersion) -t $(System.DefaultWorkingDirectory)/edge-modules/api-proxy-module/docker/manifest.yaml.template -n microsoft --tags "$(tags)"
+    - script: scripts/linux/buildManifest.sh -r $(registry.address) -v $(buildVersion) -t $(System.DefaultWorkingDirectory)/edge-modules/api-proxy-module/docker/manifest.yaml.template -n microsoft --tags "$(tags)"
```

Comment:
As a good practice, please put `"` around each of the argument provided.

Reply:
I will keep this in mind next time. But I don't think I should prioritize fixing this right now. Also, if we want to handle it everywhere all at once we can make a separate PR (since this is the PR for a cherry pick).